src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -504,12 +516,85 @@ The following diagram illustrates a simple **multi-buffer** material set example
       # +--------+--------+--------+
       #
 
+      matsets:
+        matset:
+          topology: topology
+          volume_fractions:
+            a:
+              values: [0, 0, 0, a1, 0, a0]
+              indices: [5, 3]
+            b:
+              values: [0, b0, b2, b1, 0]
+              indices: [1, 3, 2]
+
+
+Material Set Indexing Variants
+=================================
+
+Another dimension along which material sets can vary is in how their volume fractions/topological element association is established.
```

Comment:
I'd simplify and rephrase to "Material sets can also vary in how volume fractions are associated with topological elements."  I don't think we need to talk about "another dimension" since you haven't talked about the first dimension.